proof/invariant-abstract/X64/ArchInvariants_AI.thy

```diff
@@ -1051,6 +1051,17 @@ definition
 where
   "valid_x64_irq_state irqState \<equiv> \<forall>irq > maxIRQ. irqState irq = IRQFree"
 
+definition ioport_control_unique_2 :: "(cslot_ptr \<Rightarrow> cap option) \<Rightarrow> bool" where
```

Comment:
might be good to have a comment here about what this means and its intent/use for crossing to Refine

Reply:
Also, for my own curiosity since I wasn't a part of this: how did you come up with this as the necessary invariant?

Reply:
it's the same invariant as what used to be in Refine minus the bits that are already proved elsewhere in AInvs

Reply:
(comment added)